modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscription.java

```diff
@@ -386,7 +386,7 @@ public String getValue(AndesSubscription sub, QueryOptions queryOptions) {
             ("routingKey") {
         @Override
         public String getValue(AndesSubscription sub, QueryOptions queryOptions) {
-            return sub.getStorageQueue().getMessageRouterBindingKey();
+            return sub.getStorageQueue().getMessageRouterBindingKey().toLowerCase();
```

Comment:
We have few bugs recorded under MB since our message delivery is not case sensitive. Are we completely neglecting case sensitivity from MB or shouldn't we keep the possiblity for it to be fixed?